testsuite/tests/mgc/reconciliation/test_same_target.py

```diff
@@ -0,0 +1,49 @@
+"""Tests that DNSPolicy/TLSPolicy is rejected when the Gateway already has a policy of the same kind"""
+
+import pytest
+from openshift_client import selector
+
+from testsuite.policy.dns_policy import DNSPolicy
+from testsuite.tests.mgc.reconciliation import tls_policy
+
+pytestmark = [pytest.mark.mgc]
+
+
+@pytest.fixture(scope="module")
+def base_domain(hub_openshift):
+    """Returns preconfigured base domain"""
+    zone = selector("managedzone/aws-mz", static_context=hub_openshift.context).object()
+    return zone.model["spec"]["domainName"]
```

Comment:
Is it possible to use pytest `indirect=True` here instead (same as in limitador/letsencrypt tests)? Something like `@pytest.mark.parametrize("base_domain", ["aws-mz"], indirect=True)`

Reply:
In the very near future, base domain will have no parametrization at all, so this will get resolved then